packages/hxd/hxd.0.2.0/opam

```diff
@@ -26,9 +26,9 @@ depends: [
   "angstrom" {>= "0.14.0"}
   "stdlib-shims"
   "rresult"
-  "fmt"
+  "fmt"      {>= "0.8.0" & < "0.8.7"}
   "cmdliner"
-  "fpath"
+  "fpath"    {>= "0.7.3"}
```

Comment:
There is the error about `fpath` and `hxd`:
https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/f4e201536bcb0fc27c0a62d1cebd40d1323cc028/variant/opam-2.1,compilers,4.09,dkim-mirage.0.3.0,lower-bounds

Reply:
from this log, i can see that the problem is the `result` library not `fpath`.
The correct fix is thus:
```
conflicts: [
  "result" {< "1.5"}
]
```
because before result 1.5: `Result.result != Stdlib.result`